postgres-appliance/configure_spilo.py

```diff
@@ -188,15 +188,10 @@ def deep_update(a, b):
     on_restart: {{CALLBACK_SCRIPT}}
     on_role_change: {{CALLBACK_SCRIPT}}
  {{/CALLBACK_SCRIPT}}
+{{#USE_WALE}}
   create_replica_method:
-    {{#USE_WALE}}
     - wal_e
     - basebackup_fast_xlog
-    {{/USE_WALE}}
-    {{^USE_WALE}}
-    - basebackup
```

Comment:
Patroni will default to basebackup/pg_basebackup if there is no `create_replica_method:` defined in the config.